### ISSUE TITLE
[UX] Load/Save tasks - what the heck is the file extension?

### ISSUE THREAD

**Issue by pcbaer** [imapp-pl/golem-docs#7]
The Load/Save dialog does not indicate what file extension to use. When I save a task, nothing is added as extension. So when I try to load the task, the dialog shows no file available.

**Comment by magdasta** [comment]
Yeah, we are aware of this. A temporary solution is to add the '.gt' extension manually when saving the task.

**Comment by pcbaer** [comment]
Will ".gt" be the final extension?

**Comment by magdasta** [comment]
No, it won't be the final extension
